internal/tss/utils/helpers.go

```diff
@@ -3,8 +3,25 @@ package utils
 import (
 	"github.com/stellar/go/keypair"
 	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/wallet-backend/internal/tss"
 )
 
+func PayloadTOTSSResponse(payload tss.Payload) tss.TSSResponse {
+	response := tss.TSSResponse{}
+	response.TransactionHash = payload.TransactionHash
+	if payload.RpcSubmitTxResponse.Status.Status() != "" {
+		response.Status = string(payload.RpcSubmitTxResponse.Status.Status())
```

Comment:
This `.Status()` method already returns a string, so you don't need to parse it here
```suggestion
		response.Status = payload.RpcSubmitTxResponse.Status.Status()
```